kext/FileNVRAM/Version.h

```diff
@@ -0,0 +1,16 @@
+//
+//  Version.h
+//  FileNVRAM
+//
+//  Created by Micky1979 on 20/02/17.
+//  Copyright (c) 2017 xZenue LLC. All rights reserved.
+//
+
+#ifndef FileNVRAM_Version_h
+#define FileNVRAM_Version_h
+
+#define FileNVRAM_VERSION "6.9.5"
```

Comment:
For the Version.h change, please use something like the following for grabbing the version number (as a string) instead of hard coding the value:
            extern kmod_info_t kmod_info;
            kmod_info.version <-- this is where the version string is stored.